src/evidently/analyzers/stattests/registry.py

```
@@ -0,0 +1,37 @@
+from typing import List, Union, Callable, Tuple, Dict
+
+REGISTERED_STAT_TEST: Dict[str, Dict[str, Callable]] = {}
+
+
+def stattest(name: str, allowed_feature_types: List[str]):
+    def wrapper(func):
+        REGISTERED_STAT_TEST[name] = {feature_type: func for feature_type in allowed_feature_types}
```

Do we want to raise an error for collisions in test names?